### PR TITLE
Added three endpoints for facia press. 

### PR DESCRIPTION
**Pull request by jennysivapalan**
Following from #https://github.com/guardian/frontend/pull/10464 this will help identify if it is the cron schedule or the task itself that is causing the increase in thread count.

### CODE REVIEW

File: admin/conf/routes [hunk 1]
```diff
@@ -26,6 +26,9 @@ GET         /admin
 
 #Facia Press
 GET         /press/all                                                                              controllers.FrontPressController.queueAllFrontsForPress()
+GET         /press/high                                                                             controllers.FrontPressController.queueHighFrequencyFrontsForPress()
+GET         /press/standard                                                                         controllers.FrontPressController.queueStandardFrequencyFrontsForPress()
+GET         /press/low                                                                              controllers.FrontPressController.queueLowFrequencyFrontsForPress()
```
Comment (janua) [inline admin/conf/routes]: `POST`?

**Comment by janua** [comment]
:+1: Apart from POST. Can you change my endpoint to `POST` aswell? :smiley:

**Comment by jennysivapalan** [comment]
Will do :postbox:

**Comment by jennysivapalan** [comment]
@janua I added the POST endpoints and refactored some code. Please could you review?https://github.com/guardian/frontend/commit/d6c88c1e8b4205625eee9e4fd8ec6c1b0e817825

**Comment by janua** [comment]
Great! :+1:

**Pull request merged**